src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/IDecrypter.java

```diff
@@ -0,0 +1,12 @@
+package it.gov.pagopa.rtd.ms.rtdmsdecrypter.service;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import it.gov.pagopa.rtd.ms.rtdmsdecrypter.model.BlobApplicationAware;
+
+public interface IDecrypter {
+  BlobApplicationAware decrypt(BlobApplicationAware fileName);
+
+  void decryptFile(InputStream input, OutputStream output);
```

Comment:
This method as far as I understand is public just for the sake of testing. Should not be removed from the interface, and made the concrete method in the implementation class protected? What do you think?

Reply:
I agree. The method `decryptFile` should not be part of the interface